#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_disable-tests-that-need-network-access.dpatch by Jan-Pascal van Best <janpascal@vanbest.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable tests that need network access, because they fail for (some of)
## DP: the autobuilders

@DPATCH@
diff -urNad solr~/src/test/org/apache/solr/common/util/ContentStreamTest.java solr/src/test/org/apache/solr/common/util/ContentStreamTest.java
--- solr~/src/test/org/apache/solr/common/util/ContentStreamTest.java	2009-07-02 09:27:43.000000000 +0200
+++ solr/src/test/org/apache/solr/common/util/ContentStreamTest.java	2009-07-02 09:27:44.000000000 +0200
@@ -55,7 +55,7 @@
     assertTrue( IOUtils.contentEquals( new FileReader(      file ), stream.getReader() ) );
   }
   
-
+/* Disabled for Debian build because this test needs network access
   public void testURLStream() throws IOException 
   {
     String content = null;
@@ -87,4 +87,5 @@
     stream = new ContentStreamBase.URLStream( url );
     assertTrue( IOUtils.contentEquals( new StringReader( content ), stream.getReader() ) );
   }
+*/
 }
diff -urNad solr~/src/test/org/apache/solr/servlet/SolrRequestParserTest.java solr/src/test/org/apache/solr/servlet/SolrRequestParserTest.java
--- solr~/src/test/org/apache/solr/servlet/SolrRequestParserTest.java	2009-07-02 09:27:43.000000000 +0200
+++ solr/src/test/org/apache/solr/servlet/SolrRequestParserTest.java	2009-07-02 09:27:44.000000000 +0200
@@ -95,7 +95,7 @@
     }
   }
   
-
+/* Disabled for Debian builds because this test needs network access
   public void testStreamURL() throws Exception
   {
     boolean ok = false;
@@ -121,7 +121,7 @@
     assertEquals( 1, streams.size() );
     assertEquals( txt, IOUtils.toString( streams.get(0).getStream() ) );
   }
-  
+*/ 
   public void testUrlParamParsing()
   {
     String[][] teststr = new String[][] {
